src/applications/personalization/profile/util/contact-information/formValues.js

```diff
@@ -0,0 +1,89 @@
+import { ADDRESS_POU, FIELD_NAMES, USA } from '@@vap-svc/constants';
+import ADDRESS_DATA from '~/platform/forms/address/data';
+import {
+  addresses,
+  phoneNumbers,
+} from '~/applications/personalization/profile/util/contact-information/getContactInfoFieldAttributes';
+import pickBy from 'lodash/pickBy';
+
+/**
+ * Returns a copy of the input object with an added `view:livesOnMilitaryBase`
```

Comment:
deja vu... didn't we fix this incorrect comment in your previous PR?

Reply:
We did, thanks for catching that - this must have been its way through my merge conflicts